internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -609,6 +611,10 @@ func (s *service) DeleteStorageSpace(ctx context.Context, req *provider.DeleteSt
 		return &provider.DeleteStorageSpaceResponse{
 			Status: st,
 		}, nil
+	} else if len(spaces) != 1 {
```

Comment:
```suggestion
	} else if len(spaces) == 1 {
```
If we need to handle `len(spaces) > 1` we should a separate case for it.